test/services/SessionServiceSpec.scala

```diff
@@ -74,6 +75,16 @@ class SessionServiceSpec extends TestSupport with MockUIJourneySessionDataReposi
           result shouldBe true
         }
       }
+
+      "setMultipleMongoData method" should {
+        "return a future boolean value" in {
+          updateMultipleData()
+          val result: Either[Throwable, Boolean] = TestSessionService.setMultipleMongoData(Map("key" -> "value"),
```

Comment:
worth testing an error case + setMultipleMongoData with multiple values in it?

Reply:
done